include/openssl/ssl.h

```diff
@@ -541,7 +539,7 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
  */
 
 unsigned long SSL_CTX_get_options(const SSL_CTX *ctx);
-unsigned long SSL_get_options(const SSL* s);
+unsigned long SSL_get_options(const SSL *ssl);
```

Comment:
Please revert this chunk; ```SSL *s``` is the prevailing style here.

Reply:
OK, done.
I think (and wish) that one letter should be forbidden for non-POD type.
It 's a pain to track them in the code .
May be a future addition to Style Guideline ? ;)